modules/nodes-pods-using-example.adoc

```diff
@@ -57,7 +57,7 @@ spec:
       readOnly: true
   dnsPolicy: ClusterFirst
   imagePullSecrets:
-  - name: default-dockercfg-at06w
+  - name: default-cfg-at06w
```

Comment:
these might still be called dockercfg secrets.  not sure.

Reply:
@sjenning 
```
imagePullSecrets:
    - name: default-dockercfg-5zrhb
```

Reply:
just confirmed, this _is_ still called dockercfg  to match the upstream secret type `kubernetes.io/dockercfg`

Reply:
That is what I saw in the console.